test/input.txt

```diff
@@ -96,7 +96,7 @@
   find bet
   # does not match if none have keyword
   find 23912039120
-  # matching should be case-sensitive
+  # matching should be case-insensitive
   find betsy
```

Comment:
What do you notice about the way that the tests are ordered in `input.txt` (i.e. this block and the next block)? (Hint: One set seems to be for invalid inputs...)

Reply:
Oh right! I just realised.. in this case, I could probably add more tests to show the case insensitive search is comprehensive?